src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1739,42 +1739,47 @@ private static SyntaxToken GetImplicitConstructorBodyToken(CSharpSyntaxNode cont
             }
         }
 
-        internal static BoundCall GenerateObjectConstructorInitializer(MethodSymbol constructor, DiagnosticBag diagnostics)
+        internal static BoundCall GenerateBaseConstructorInitializer(MethodSymbol constructor, DiagnosticBag diagnostics)
```

Comment:
> not sure what part of it was reflecting the (parameter-less) part. 

The Object part, we know it mast have a parameter-less constructor. The suggested name looks fine to me.

---
In reply to: [115099517](https://github.com/dotnet/roslyn/pull/18715#discussion_r115099517) [](ancestors = 115099517,115095529)